modules/vulnerabilities/unix/http/jenkins_cli/secgen_metadata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<vulnerability xmlns="http://www.github/cliffe/SecGen/vulnerability"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.github/cliffe/SecGen/vulnerability">
+
+  <name>Jenkins CLI Deserialization - v2.56 and below</name>
+  <author>James Davis</author>
+  <module_license>MIT</module_license>
+  <description>An unauthenticated Java object deserialization vulnerability exists
+    in the CLI component for Jenkins versions `v2.56` and below.
+    The `readFrom` method within the `Command` class in the Jenkins
+    CLI remoting component deserializes objects received from clients without
+    first checking / sanitizing the data. Because of this, a malicious serialized
+    object contained within a serialized `SignedObject` can be sent to the Jenkins
+    endpoint to achieve code execution on the target.</description>
+
+  <type>http</type>
+  <type>bruteforceable</type>
+  <type>in_the_wild</type>
+  <privilege>user_rwx</privilege>
+  <access>remote</access>
+  <platform>linux</platform>
+  <difficulty>low</difficulty>
+
+  <read_fact>port</read_fact>
+  <read_fact>strings_to_leak</read_fact>
+  <read_fact>leaked_filenames</read_fact>
+
+  <default_input into="port">
+    <generator module_path=".*/random_unregistered_port" />
+  </default_input>
+
+  <default_input into="strings_to_leak">
+    <generator type="flag_generator" />
```

Comment:
Minor, but except where it's specifically a CTF puzzle challenge, we avoid having flags as the default